libs/blocks/table/table.js

```diff
@@ -113,7 +116,7 @@ function handleHighlight(table) {
     firstRow.classList.add('row-heading');
   }
 
-  handleHeading(headingCols, isPriceBottom);
+  handleHeading(headingCols, isPriceBottom, isMerch);
```

Comment:
No real need to create a const for `isMerch` (or `isPriceBottom`), you can directly pass the condition; however, you don't need to change the code for this small thing.

Reply:
@narcis-radu I have passed the table reference and used the classlist comparison directly in place of passing the value.